client/app/scripts/vendor/term.js

```diff
@@ -2756,6 +2756,13 @@ Terminal.prototype.keyDown = function(ev) {
     case 123:
       key = '\x1b[24~';
       break;
+    // Special case for US-intl '/"
+    case 222:
+      key = '\'';
+      if (ev.shiftKey) {
+        key = '"';
+      }
+      break;
```

Comment:
We're lucky, the library is unmaintained! :D

We'd probably do something like `patch -p1 < git log -p term.js`.

We could look into forking the lib on github which I guess is the proper
way to do it..

On 7 July 2016 at 16:10, Alfonso Acosta <notifications@github.com> wrote:

> In client/app/scripts/vendor/term.js
> <https://github.com/weaveworks/scope/pull/1647#discussion_r69913191>:
>
> > @@ -2756,6 +2756,13 @@ Terminal.prototype.keyDown = function(ev) {
> >      case 123:
> >        key = '\x1b[24~';
> >        break;
> > +    // Special case for US-intl '/"
> > +    case 222:
> > +      key = '\'';
> > +      if (ev.shiftKey) {
> > +        key = '"';
> > +      }
> > +      break;
>
> I don't know how you are handling vendoring but, How do you make sure
> changes made directly are not lost when updating the library?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/weaveworks/scope/pull/1647/files/d506429d8fed34c310e481dbf5e1670fdceabc51#r69913191>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AABV-AfOKok3RUMynE4CCJS20GchAEqbks5qTQjXgaJpZM4JF5v9>
> .
>